package.json

```diff
@@ -246,9 +245,12 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^23.6.0",
     "benv": "^3.3.0",
+    "brotli-webpack-plugin": "^1.1.0",
     "cache-loader": "^1.2.2",
     "coffee-loader": "^0.8.0",
+    "compression-webpack-plugin": "^2.0.0",
     "core-js": "^2.5.7",
+    "crypto": "^1.0.1",
```

Comment:
This package is deprecated. It has been included in Node >= v7, so you can just import it directly. Can you remove this package dep?